src/components/button/button.scss

```diff
@@ -44,9 +44,11 @@
 }
 
 // Only flat buttons and icon buttons (not raised or fabs) have a hover style.
-[md-button]:hover, [md-icon-button]:hover {
-  // Use the same visual treatment for hover as for focus.
-  @include md-button-focus();
+[md-button], [md-icon-button] {
+  &:not([disabled]):hover {
```

Comment:
Done, although I think that this adds a little to the selector length, but reduces specificity. It seemed strange that the `:after` is used to add the background.